docs/ref/models/fields.txt

```diff
@@ -1185,6 +1185,52 @@ when :attr:`~django.forms.Field.localize` is ``False`` or
     information on the difference between the two, see Python's documentation
     for the :mod:`decimal` module.
 
+``GeneratedField``
+------------------
+
+.. versionadded:: 5.0
+
+.. class:: GeneratedField(expression, output_field=None, db_persist=True, **kwargs)
+
+A field which is computed based on other fields in the model. Similar to a
+database view, this field is managed and updated by the database itself. Uses
+the ``GENERATED`` SQL syntax.
+
+There are two kinds of generated columns: stored and virtual. A stored generated
+column is computed when it is written (inserted or updated) and occupies storage
+as if it were a normal column. A virtual generated column occupies no storage and
+is computed when it is read. Thus, a virtual generated column is similar to a view
+and a stored generated column is similar to a materialized view.
+
+``GeneratedField`` is supported on MariaDB, MySQL, PostgreSQL, SQLite, and Oracle
```

Comment:
```suggestion
``GeneratedField`` is supported on MariaDB, MySQL, PostgreSQL, SQLite, and Oracle.
```

In ma latest suggestion I've added "Oracle" in this list but I've forgot the final dot.